geometry_msgs/Vector3[] dir_1
geometry_msgs/Vector3[] dir_2
geometry_msgs/Vector3[] dir_3
geometry_msgs/Vector3[] centers
parameters[] magnitudes
geometry_msgs/Vector3[] cyl_p1
geometry_msgs/Vector3[] cyl_p2
float64[] radius


